docs/DeveloperGuide.adoc

```diff
@@ -729,8 +729,62 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user with many people in the address book |list all people in address book |see whose contacts the address book already has
+
+|`* * *` |user |clear all people in the address book |remove all contacts conveniently
+
+|`* * *` |user |list the command history |view all commands already entered
+
+|`* * *` |user |undo previous command |amend any mistakes
+
+|`* * *` |user |redo the previously undone command |amend any mistakes
+
+|`* * *` |user |edit a person's information |update changes to their particulars
+
+|`* * *` |user with many people in the address book |make relation links between contacts |link up contacts that have connections with each other
+
+|`* * *` |user with many people in the address book |sort contacts by different fields in alphabetical order |arrange them according to the fields
+
+|`* * *` |user with many people in the address book |sort contacts to my preference |arrange them according to their importance / significance
+
+|`* * *` |user |find users based on tags |locate different groups easily
+
+|`* * *` |user |rename an existing tag |update a tag which is no longer suitable to describe the people who have them
+
+|`* * *` |user |run the executable file of the address book |access the application conveniently
+
+|`* *` |user with many people in the address book |delete people in address book by tags |remove people of the same group
+
+|`* *` |user with many people in the address book |change the view mode of GUI |display different number of contacts per page
```

Comment:
Shouldn't it be in NFR? I would argue that `listing all contacts` is a feature, whereas how you display the listed contacts is not really a feature.